docs/docs/troubleshooting/hardware-issues.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Hardware Issues
+sidebar_label: Hardware Issues
+description: Troubleshooting common hardware issues.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+export const TesterTabs = (props) => (
```

Comment:
I think this page wants a small blurb at the top. Essentially, it can say that these are a set of non-comprehensive recommendations for identifying and solving common hardware problems encountered with DIY keyboards (intended to run ZMK?).

In addition, I am not sure how to exactly word this right now, but I think it's important to note that this is in an advisory capacity only. Maybe I am being too careful but I basically don't want users to think that ZMK project provides official hardware support of any kind. To that end, we should also urge users to contact the keyboard seller or manufacturer for any issues with non-DIY keyboards.

Reply:
I agree, forgot about making that note. Added a warning block.